config/crd/bases/scheduling.x-k8s.io_podgroups.yaml

```diff
@@ -4,7 +4,7 @@ kind: CustomResourceDefinition
 metadata:
   annotations:
     api-approved.kubernetes.io: https://github.com/kubernetes-sigs/scheduler-plugins/pull/50
-    controller-gen.kubebuilder.io/version: v0.16.5
+    controller-gen.kubebuilder.io/version: v0.18.0
```

Comment:
`v0.17.3` seems to be the "exact" match: https://github.com/kubernetes-sigs/controller-tools/blob/v0.17.3/go.mod#L16